build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    java
+    id("org.jetbrains.intellij") version "0.7.2"
+}
+
+intellij {
+    type = "IC"
+    version = "2021.1"
```

Comment:
let's update to 2021.2 and check if everything works fine

Reply:
everything is not fine(